<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <widget class="BotonAnimado" name="btnJugar">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>160</y>
          <width>132</width>
          <height>36</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 179, 31)</string>
        </property>
        <property name="text">
         <string>Jugar</string>
        </property>
       </widget>
       <widget class="BotonAnimado" name="btnConfigurar">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>240</y>
          <width>161</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Configurar</string>
        </property>
       </widget>
       <widget class="BotonAnimado" name="btnAyuda">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>320</y>
          <width>132</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Ayuda</string>
        </property>
       </widget>
       <widget class="BotonAnimado" name="btnSalir">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>400</y>
          <width>132</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BotonAnimado</class>
   <extends>QPushButton</extends>
   <header location="global">botonanimado.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
